spec/tasks/cleanup_orphaned_users_spec.rb

```diff
@@ -22,7 +18,7 @@
     it "it finds users without roles" do
       expect {
         task.invoke
-      }.to output(/These users have no roles: \[#{admin_user.id}, #{team_member_user.id}]/).to_stdout
+      }.to output(/These users have no roles: \[#{admin_user.id}, #{team_member_user.id}\]/).to_stdout
```

Comment:
fixed a warning: this regex was missing an escaping \ before it's ], which is invalid syntax but was still able to be parsed